recaptcha_enterprise/snippets/pom.xml

```diff
@@ -80,6 +94,16 @@
       <artifactId>webdrivermanager</artifactId>
       <version>5.5.3</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents.client5</groupId>
+      <artifactId>httpclient5</artifactId>
+      <version>5.2.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-http-jdk-client</artifactId>
+      <version>4.8.1</version>
+    </dependency>
```

Comment:
nit: these dependencies seem to be intended for testing. please, add the `test` [scope].

[scope]: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope

Reply:
Done.